packages/composite-checkout/src/lib/payment-methods/stripe-credit-card-fields.js

```diff
@@ -340,7 +340,7 @@ const StripeFieldWrapper = styled.span`
 		font-size: 16px;
 		box-sizing: border-box;
 		border: 1px solid
-			${( props ) => ( props.hasError ? props.theme.colors.error : props.theme.colors.borderColor) };
+			${( props ) => ( props.hasError ? props.theme.colors.error : props.theme.colors.borderColor )};
```

Comment:
I would've expected space in here: `${( props ) => ( props.hasError ? props.theme.colors.error : props.theme.colors.borderColor )};
`